### ISSUE TITLE
The "Search Search" button - tiny screenreader annoyance

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#616]
This might be a higher-level Blacklight thing, but a screenreader reveals that the Search button has redundant text, reading both the button text `Search` and the icon title `Search`. 

Even when the button text is not visible, it's still read by the SR, so I think the icon could be safely `aria-hidden`.
